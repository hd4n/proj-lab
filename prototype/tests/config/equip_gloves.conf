
@fields
em e1

@items
gl g1

@virologists
v1 e1