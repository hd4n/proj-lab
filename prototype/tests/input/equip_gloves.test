load equip_gloves.conf
equip v1 g1
getstats v1