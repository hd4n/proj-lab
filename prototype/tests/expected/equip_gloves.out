Load: sikeres
equip: v1 g1

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_g1
	ef_null
